lib/pf/file_paths.pm

```diff
@@ -330,6 +332,8 @@ $portal_modules_config_file = catfile($conf_dir,"portal_modules.conf");
 $portal_modules_default_config_file = catfile($conf_dir,"portal_modules.conf.defaults");
 $pfmon_config_file = catfile($conf_dir,"pfmon.conf");
```

Comment:
Understand that this line is still here for the upgrade script but can we remove it and use the full path in the upgrade script ?